src/main/java/edu/harvard/iq/dataverse/ingest/IngestServiceBean.java

```diff
@@ -1269,7 +1269,7 @@ public boolean extractMetadataNcml(DataFile dataFile, Path tempLocationPath) {
                     String ncml = netcdfFile.toNcml("");
                     inputStream = new ByteArrayInputStream(ncml.getBytes(StandardCharsets.UTF_8));
                 } else {
-                    logger.info("NetcdfFiles.open() could not open file id " + dataFile.getId() + " (null returned).");
+                    logger.fine("NetcdfFiles.open() could not open file id " + dataFile.getId() + " (null returned).");
```

Comment:
@pdurbin I'm seeing that @rtreacy has changed the log level to .fine here; so this should remove the noisy message. What confuses me though is why are we trying to run this `extractMetadataNcml()` methods on ALL files, of all types here in the first place? I mean, you do already know which files are netcdf and which are not at this point. Because you have a dedicated test for this format in `FileUtil.determineFileType()`, which actually does the same thing - `NetcdfFiles.open()` - and gives the file the correct mime type `application/netcdf` or `.../x-hdf5` if that works. 
So you should definitely have one of the 2 types above in `dataFile.getContentType()` at this point for any file that can be opened as netcdf. So unless I'm missing something, it seems like this netcdf extraction should only be performed on those types. Just like we do with fits extraction - ?

Reply:
(I only noticed this because I was deep in `FileUtil.determineFileType()` myself, on account of the runaway file descriptors)

Reply:
@landreev yes, good point. I just created this PR as an alternative using your idea of only running `extractMetadataNcml()` on NetCDF or HDF5 files:

- #9510